src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -44,11 +49,16 @@ public class PersonCard extends UiPart<Region> {
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public PersonCard(Person person, int displayedIndex) {
+    public PersonCard(Person person, BigFraction bal, int displayedIndex) {
         super(FXML);
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
+        String balString = FractionUtil.toString(bal, 2);
+        if (bal.signum() > 0) {
```

Comment:
Can consider abstact this adding `+` into the method in `FractionUtil` instead of here.